src/sentry/integrations/github/client.py

```diff
@@ -221,6 +189,27 @@ def _populate_trees(self, repositories: List[Dict[str, str]]) -> Dict[str, RepoT
         For every repository, fetch the tree associated and cache it.
         This function takes API rate limits into consideration to prevent exhaustion.
         """
+
+        def process_error(error: ApiError, extra: Dict[str, str]) -> None:
+            msg = "Continuing exectuion."
```

Comment:
`execution`

This is easier to read if we just include it as part of the logger messages below since it's not conditional